Bottom: 7da45b713b0738e99beef02e60d981d284446da7
Top:    802db68ae6709d9a858d7719ecefe270aea62ab5
Author: Igor Stoppa <igor.stoppa@huawei.com>
Date:   2018-06-07 16:06:51 +0300

Use designated initializer for struct list_head in list.h


---

diff --git a/include/linux/list.h b/include/linux/list.h
index 4b129df4d46b..11847c96571f 100644
--- a/include/linux/list.h
+++ b/include/linux/list.h
@@ -18,7 +18,10 @@
  * using the generic single-entry routines.
  */
 
-#define LIST_HEAD_INIT(name) { &(name), &(name) }
+#define LIST_HEAD_INIT(name) {	\
+	.next = &(name),	\
+	.prev = &(name),	\
+}
 
 #define LIST_HEAD(name) \
 	struct list_head name = LIST_HEAD_INIT(name)
